qtp_diversity/summary.py

```diff
@@ -139,10 +139,53 @@ def _generate_alpha_vector_summary(files, metadata, out_dir):
     return html_fp, html_dir
 
 
+def _generate_feature_data_taxonomy(files, metadata, out_dir):
+    # Magic number [0] -> there is only one plain text file and it is the
+    # feature data taxonomy
+    fdt_fp = files['plain_text'][0]
+
+    if 'qza' not in files:
+        fdt_qza = join(out_dir, 'taxonomy.qza')
+        # Get the SampleData[AlphaDiversity] qiime2 artifact
+        cmd = ('qiime tools import --input-path %s --output-path %s '
+               '--type "FeatureData[Taxonomy]"'
+               % (fdt_fp, fdt_qza))
+        std_out, std_err, return_value = system_call(cmd)
+        if return_value != 0:
```

Comment:
It might be a good idea to use https://docs.python.org/3.6/library/errno.html#errno.errorcode to convert the return code to a string value that can help decipher the problem, since things can fail for a lot of reasons.